apps/vast/README.md

```diff
@@ -0,0 +1,29 @@
+Threat Bus to VAST Bridge
+=========================
+
+Threat Bus is a publish-subscribe broker for threat intelligence. It is expected
+that applications register themselves at the bus. Since vast cannot do so on its
+own (yet), the `vast-bridge.py` is used for that purpose.
```

Comment:
```suggestion
Threat Bus is a publish-subscribe broker for threat intelligence. It is expected
that applications register themselves at the bus. Since VAST cannot do so on its
own (yet), `vast-bridge.py` implements that functionality in the meantime.
```